classes/asserters/phpResource.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace mageekguy\atoum\asserters;
+
+use
+	mageekguy\atoum\asserters
+;
+
+class phpResource extends asserters\variable
+{
+	public function setWith($value)
+	{
+		parent::setWith($value);
+
+		if ($this->analyzer->isResource($this->value) === true)
+		{
+			$this->pass();
+		}
+		else
+		{
+			$this->fail($this->_('%s is not a resource', $this));
+		}
+
+		return $this;
+	}
+
+	public function isOfType($type, $failMessage = null)
```

Comment:
What to do with space or hyphen? For instance “persistent cubrid connection” is transformed into…?